content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -638,6 +639,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `KubeletPodResources`: Enable the kubelet's pod resources gRPC endpoint. See
   [Support Device Monitoring](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/606-compute-device-assignment/README.md)
   for more details.
+- `KubeletPodResourcesGetAllocatable`: Enable the kubelet's pod resources `GetAllocatableResources` functionality. See
+  [Allocatable Resources Reporting](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2403-pod-resources-allocatable-resources/README.md)
+  for more details.
```

Comment:
@fromanirh we prefer _not_ to link to KEPs when explaining feature gates. Instead, write a short summary in a concept or task page, and hyperlink there.

Reply:
sorry I missed this previously. I think a good solution is to link back to https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources, and I'll change accordingly.